fujiji-server/src/app.js

```diff
@@ -33,17 +27,23 @@ app.get('/appstatus', async (req, res) => {
   const [allToken] = await sequelize.query('SELECT * FROM fujiji_token');
   console.log(allToken);
 
-  return res.status(200).json({ response, allUser, allListing, allToken });
+  return res.status(200).json({
+    response, allUser, allListing, allToken,
+  });
 });
 
-app.listen(port, async () => {
-  console.log(`Server is up on port ${appUrl}:${port}`);
-  console.log('test');
-
-  try {
-    await sequelize.authenticate();
-    console.log('Connection has beena established successfully.');
-  } catch (error) {
-    console.error('Unable to connect to the database:', error);
-  }
-});
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
Is this necessary? I'm not 100% sure if this part is going to be different between `dev` and `prod` environment

Reply:
If I don't put that, it throws an error when I run `npm test`. So I tried doing what someone did here https://stackoverflow.com/questions/54422849/jest-testing-multiple-test-file-port-3000-already-in-use

Reply:
Ah okay! that's a tricky one